packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js

```diff
@@ -0,0 +1,62 @@
+import { Markdown, Text } from 'grommet'
+import { inject, observer, PropTypes as MobXPropTypes } from 'mobx-react'
+import { observable } from 'mobx'
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+function storeMapper (stores) {
+  const {
+    addAnnotation
+  } = stores.classifierStore.classifications
+  const annotations = stores.classifierStore.classifications.currentAnnotations
+  return {
+    addAnnotation,
+    annotations
+  }
+}
+
+@inject(storeMapper)
+@observer
+class Graph2dRangeXTask extends React.Component {
+  onChange (index, event) {
+    const { addAnnotation, task } = this.props
+    if (event.target.checked) addAnnotation(index, task)
+  }
+
+  render () {
+    const {
+      annotations,
+      task
+    } = this.props
+    let annotation
+    if (annotations && annotations.size > 0) {
+      annotation = annotations.get(task.taskKey)
+    }
+    return (
+      <Text size='small' tag='legend'>
+        <Markdown>
+          {task.instruction}
+        </Markdown>
+      </Text>
+    )
+  }
+}
+
+Graph2dRangeXTask.wrappedComponent.defaultProps = {
+  addAnnotation: () => {},
+  annotations: observable.map(),
+  task: {}
+}
+
+Graph2dRangeXTask.wrappedComponent.propTypes = {
+  addAnnotation: PropTypes.func,
+  annotations: MobXPropTypes.observableMap,
```

Comment:
I didn't know about this - v cool!

Reply:
I actually took this from the `DrawingTask.js` code - I thought you introduced this? 😮